applications/nrf_desktop/src/modules/hid_forward.c

```diff
@@ -80,7 +82,7 @@ static void forward_hid_report(uint8_t report_id, const uint8_t *data, size_t si
 
 	k_spinlock_key_t key = k_spin_lock(&lock);
 
-	if (!usb_ready) {
+	if (!usb_ready && !suspended && !enqueued_report_count) {
```

Comment:
I have added it to allow system to pass the wakeup button to the USB. It actually should work as reports will be enqueued from the moment standby happens (this is when USB unsubscribes). USB will disconnect but this will not clear state. When USB is active again and subscribes it should get enqueued reports. There are some potential problems here but if woken up from keyboard in normal situation this should pass the button.

Reply:
Ok, let's leave it then.